az_quickstart.install

```diff
@@ -226,3 +226,35 @@ function az_quickstart_update_920103() {
     return t('Environment Indicator module installed.');
   }
 }
+
+/**
+ * Transition bootstrap_barrio variables into az_barrio.
+ */
+function az_quickstart_update_920104() {
```

Comment:
Assuming we want this to be a 2.6 minor release only change, I think we'll want this to be `920601`:
```suggestion
function az_quickstart_update_920601() {
```